utils/irqbalance/files/irqbalance.init

```diff
@@ -33,11 +33,23 @@ start_service() {
 	}
 	config_list_foreach irqbalance banirq handle_banirq_value
 
+	# debug
```

Comment:
Do we need this comment?

Reply:
No. Removed and rebased the whole thing.